pants-plugins/tests/python/internal_backend_test/sitegen/test_sitegen.py

```diff
@@ -191,6 +193,7 @@ def test_transforms_not_discard_page_tocs(self):
     self.assertIn('DEPTH=1 LINK=one TEXT=Section One', rendered)
     self.assertIn('DEPTH=1 LINK=two TEXT=Section Two', rendered)
 
+  @pytest.mark.skipif(sys.version_info >= (3,0), reason="TODO: See #6062.")
```

Comment:
What is the general philosophy on skipping py3 test given 
```
        # Allowed to fail.
        - ./build-support/bin/ci.sh -x -3efkmrjcnt "${SHARD}" || exit 0
```
in .travis.yml?

Reply:
To make it past the "internal" tests in order to run the unit tests.

Reply:
Oh gotcha. Thanks!